src/sinks/datadog/logs.rs

```diff
@@ -79,8 +74,6 @@ fn encode_event(
     api_key: &str,
     encoding: &EncodingConfig<Encoding>,
 ) -> Option<Bytes> {
-    encoding.apply_rules(&mut event);
```

Comment:
Do we need to add this back somewhere?

Reply:
This one caught me too. That function calls `util::encode_event(event, encoding)` further down, which calls this, so this change just removes the double call to it.

Reply:
Aha, that makes sense.